src/dice/hash/blake/internal/blake3/CMakeLists.txt

```diff
@@ -0,0 +1,204 @@
+cmake_minimum_required(VERSION 3.9 FATAL_ERROR)
```

Comment:
I'm now wondering if it makes sense to do what we do in rdf4cpp and download the files from github instead of committing them here